lib/components/artist/header.js

```diff
@@ -15,9 +15,9 @@ import SerifText from '../text/serif'
 
 class Header extends React.Component {
   state: {
-    following: boolean;
-    followersCount: number;
-  };
+    following: ?boolean,
```

Comment:
we don't know at first, so it's nullable - it was being used this way, but not typed for it - so it errored